cachi2/core/package_managers/bundler/scripts/lockfile_parser.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
```

Comment:
Not a deal breaker for me, but I'd also introduce this helper script in a separate commit preceding the parser changes. That is to keep git history nice and clean and not to make the script go unnoticed, IOW you can see a commit introducing a Ruby script (that doesn't happen very often, so it's an unusual change) and another one introducing the parser.